lib/tasks/evm_application.rb

```diff
@@ -102,11 +102,12 @@ def self.output_workers_status(servers)
            w.miq_server_id,
            w.queue_name || w.uri,
            w.started_on && w.started_on.iso8601,
-           w.last_heartbeat && w.last_heartbeat.iso8601]
+           w.last_heartbeat && w.last_heartbeat.iso8601,
+           (w.proportional_set_size || w.memory_usage || 0) / 1.megabyte]
```

Comment:
Yeah, I agree with you.  Maybe blank would be better than N/A.  Let me look at that and clean it up.